src/ucp/wireup/wireup.c

```diff
@@ -1537,9 +1537,14 @@ ucs_status_t ucp_wireup_init_lanes(ucp_ep_h ep, unsigned ep_init_flags,
         UCS_BITMAP_CLEAR(&current_tl_bitmap);
         for (lane = 0; lane < ucp_ep_config(ep)->key.num_lanes; ++lane) {
             rsc_idx = ucp_ep_config(ep)->key.lanes[lane].rsc_index;
-
             UCS_BITMAP_SET(current_tl_bitmap, rsc_idx);
-            ucs_assert(UCS_BITMAP_GET(tl_bitmap, rsc_idx));
+            ucs_assertv_exp(
```

Comment:
why needed to add such macro?

Reply:
Described here: https://github.com/openucx/ucx/pull/9141#issuecomment-1600089488

Reply:
I found only [this solution](https://stackoverflow.com/a/11694231) but it doesn't work. So I have no ideas how to do that without manual expression string definition.

Reply:
try to change
```
#define ucs_assert(...)       ucs_assert_always(__VA_ARGS__)
#define ucs_assertv(...)      ucs_assertv_always(__VA_ARGS__)
```

to
```
#define ucs_assert       ucs_assert_always
#define ucs_assertv      ucs_assertv_always
```

Reply:
That one works, thanks, applied